src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/appsettings.json

```diff
@@ -31,6 +31,8 @@
     "RuntimeCookieName": "AltinnStudioRuntime",
     "SBLBaseAdress": "https://at22.altinn.cloud/",
     "RoleCacheTimeout": 5,
+    "MainUnitCacheTimeout": 5,
```

Comment:
Main/subunit relations are AFAIK only change after the nightly ER-batch (unlike keyrole parties, as DAGL etc. assignments/revocations might be expedited from ER and thus might occur at any time). We can perhaps keep the cached value around until after we know the ER batch has run? 

In any case, this can probably safely be increased to several minutes. I'm assuming that for any given user session, there are multiple calls within a short time window to authorize one or more instances for the same reportee, as the user paginates, performs searches, opens instances etc. Not having to check whether or not a reportee has a parent unit for all those calls appears to be a simple and safe optimization.

Reply:
Closig this for now pending a better overall cachine regime.